plothist/plotters.py

```diff
@@ -522,7 +522,7 @@ def plot_comparison(
 
     if np.allclose(lower_uncertainties, upper_uncertainties, equal_nan=True):
         hist_comparison = bh.Histogram(hist_2.axes[0], storage=bh.storage.Weight())
-        hist_comparison[:] = np.c_[comparison_values, lower_uncertainties ** 2]
+        hist_comparison[:] = np.c_[comparison_values, lower_uncertainties**2]
```

Comment:
We should get the same `black` version.
I had `21.10b0` and upgraded to the latest `23.11.0`.

Reply:
I updated to 23.10b0, the file stays the same as in the PR